lib/core/feature-flags/src/lib/providers/debug-feature-flags.provider.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { APP_INITIALIZER /*EnvironmentProviders, makeEnvironmentProviders*/ } from '@angular/core';
+import {
+    FlagsOverridabilityToken,
+    FeaturesServiceToken,
+    QaFeaturesHelperConfig,
+    WritableFeaturesServiceConfig,
+    WritableFeaturesServiceConfigToken,
+    WritableFeaturesServiceToken
+} from '../interfaces/features.interface';
+import { StorageFeaturesService } from '../services/storage-features.service';
+import { DebugFeaturesService } from '../services/debug-features.service';
+import { QaFeaturesHelper } from '../services/qa-features.helper';
+import { DOCUMENT } from '@angular/common';
+
+/**
+ *
+ * @param config Configuration for the Feature Flags
+ * @returns Environment Providers for Feature Flags
+ */
+export function provideDebugFeatureFlags(config: WritableFeaturesServiceConfig & QaFeaturesHelperConfig) /*: EnvironmentProviders*/ {
```

Comment:
contains commented out code

Reply:
A16 will use this format.

Reply:
I have already cleaned the code but I'll create a doc with steps to upgrade to ng16, so we don't loose this knowledge